src/cf-core/src/cf-utilities.less

```diff
@@ -33,14 +34,14 @@
 //
 
 .u-visually-hidden {
-  position: absolute;
-  width: 1px;
-  height: 1px;
-  border: 0;
-  margin: -1px;
-  padding: 0;
-  overflow: hidden;
-  clip: rect(0 0 0 0);
+    position: absolute;
+    width: 1px;
+    height: 1px;
+    border: 0;
+    margin: -1px;
+    padding: 0;
+    overflow: hidden;
+    clip: rect(0 0 0 0);
```

Comment:
Hmm… tricky one. [According to MDN](https://developer.mozilla.org/en-US/docs/Web/CSS/clip), this property is deprecated and should not be used. But [it's still present](https://github.com/h5bp/html5-boilerplate/blob/master/src/css/main.css#L126) in the HTML5 Boilerplate (from which we originally cribbed this utility).

I'm not sure if removing it would break the utility, so I think we should leave it for now. The MDN page states that both space- and comma-separated syntaxes are separate, but space-separated is more backward-compatible.